about.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css" />
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>10-Gano Courses</title>
+</head>
+<body>
+  <!-- navbar -->
+  <nav class="navbar" id="navbar">
+    <div class="logo-container">
+      <a href="index.html">
+        <img src="assets/10-gano-high-resolution-logo-black-on-transparent-background.png" alt="cc-logo" />
+      </a>
+    </div>
+    <div class="bars" id="toggle-icon" onclick="toggleMobileMenu(this)">
+      <div class="bar1"></div>
+      <div class="bar2"></div>
+      <div class="bar3"></div>
+      <ul class="nav-menu">
+        <li>
+          <a href="about.html">About</a>
+        </li>
+        <li>
+          <a>Program</a>
+        </li>
+        <li>
+          <a>Join</a>
+        </li>
+        <li>
+          <a>Sponsor</a>
+        </li>
+        <li>
+          <a>News</a>
+        </li>
+      </ul>
+    </div>
+  </div>
+  <div class="links-container">
+    <ul>
+      <li>
+        <a href="about.html">About</a>
+      </li>
+      <li>
+        <a>Program</a>
+      </li>
+      <li>
+        <a>Join</a>
+      </li>
+      <li>
+        <a>Sponsor</a>
+      </li>
+      <li>
+        <a>News</a>
+      </li>
+    </ul>
+  </div>
+</nav>
+<!-- header -->
+<header class="header">
+  <div class="about-header-container">
+    <h2 class="about-intro">10-Gano Courses</h2>
```

Comment:
- [ ] Kindly make sure your `About` page has only one main heading `h1` but not more than one. I would like to suggest changing `h2` to `h1` in order to have the man heading on the page.